frontend/app/mocks/power-platform-api.server.ts

```diff
@@ -231,6 +231,27 @@ export function getPowerPlatformApiMockHandlers() {
       return HttpResponse.json(mockBenefitApplicationResponse);
     }),
 
+    /**
+     * Handler for PATCH request to update personalInformation address to Power Platform
+     */
+    http.patch('https://api.example.com/dental-care/applicant-information/dts/v1/applicant/:sin', async ({ params, request }) => {
+      log.debug('Handling request for [%s]', request.url);
+
+      const subscriptionKey = request.headers.get('Ocp-Apim-Subscription-Key');
+      if (!subscriptionKey) {
+        return new HttpResponse('Access denied due to missing subscription key. Make sure to include subscription key when making requests to an API.', { status: 401 });
+      }
+
+      const parsedSin = z.string().safeParse(params.sin);
+      if (!parsedSin.success) {
+        throw new HttpResponse('invalid sin', { status: 404, headers: { 'Content-Type': 'text/plain' } });
```

Comment:
I think this should be a `400` response, and it might make sense to return a JSON payload (unless the *actual* PP API returns plaintext).

Reply:
fixed